src/base/http/server.cpp

```diff
@@ -62,7 +62,8 @@ void Server::enableHttps(const QSslCertificate &certificate, const QSslKey &key)
 void Server::disableHttps()
 {
     m_https = false;
-    m_certificate.clear();
+    foreach (QSslCertificate c, m_certificates)
```

Comment:
Aren't you cleaning a copy of an object from `m_certificates`? Moreover, why not just `m_certificates.clear()` call is used here?